ubuntu/python/Dockerfile

```diff
@@ -1,36 +1,26 @@
-FROM ubuntu:18.04 as builder
-
-RUN apt-get update && apt-get install --yes \
-    wget \
-    libdigest-sha-perl \
-    bzip2
-
-# Download miniconda 4.5.12, then upgrade it to 4.6.12.
-RUN wget --quiet --output-document miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-4.5.12-Linux-x86_64.sh \
-    && (echo '866ae9dff53ad0874e1d1a60b1ad1ef8  miniconda.sh' | md5sum -c) \
-    && (echo 'e5e5b4cd2a918e0e96b395534222773f7241dc59d776db1b9f7fedfcb489157a  miniconda.sh' | shasum -a 256 -c) \
-    # Conda must be installed at /databricks/conda
-    && /bin/bash miniconda.sh -b -p /databricks/conda \
-    && rm miniconda.sh \
-    && /databricks/conda/bin/conda install --name base conda=4.6.12
-
 FROM databricksruntime/minimal:latest
 
-COPY --from=builder /databricks/conda /databricks/conda
-
-COPY env.yml /databricks/.conda-env-def/env.yml
-
-RUN /databricks/conda/bin/conda env create --file /databricks/.conda-env-def/env.yml \
-    # Source conda.sh for all login shells.
-    && ln -s /databricks/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
-
-# Conda recommends using strict channel priority speed up conda operations and reduce package incompatibility problems.
-# Set always_yes to avoid needing -y flags, and improve conda experience in Databricks notebooks.
-RUN /databricks/conda/bin/conda config --system --set channel_priority strict \
-    && /databricks/conda/bin/conda config --system --set always_yes True
-
-# This environment variable must be set to indicate the conda environment to activate.
-# Note that currently, we have to set both of these environment variables. The first one is necessary to indicate that this runtime supports conda.
-# The second one is necessary so that the python notebook/repl can be started (won't work without it)
-ENV DEFAULT_DATABRICKS_ROOT_CONDA_ENV=dcs-minimal
-ENV DATABRICKS_ROOT_CONDA_ENV=dcs-minimal
+# Installs python 3.6 and virtualenv for Spark and Notebooks
+RUN apt-get update \
+  && apt-get install -y \
+    python3 \
+    virtualenv \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Initialize the default environment that Spark and notebooks will use
+RUN virtualenv -p python3 --system-site-packages /databricks/python3
+
+# These python libraries are used by Databricks notebooks and the Python REPL
+# You do not need to install pyspark - it is injected when the cluster is launched
+RUN /databricks/python3/bin/pip install \
+  six==1.15.0 \
+  ipython==7.16.0 \
+  numpy==1.19.5 \
+  pandas==1.1.5 \
+  pyarrow==4.0.0 \
+  matplotlib==3.3.4 \
+  jinja2==2.11.3
```

Comment:
Could you document which package versions are different from their versions in DBR 9.0?
Just want to make sure the next time someone works on this, it's clear that the difference in versions is intentional.

Reply:
Ok so following up from your other comment, it seems bumping to python 3.8 now allows pip to find these packages! I'll add a comment mentioning that these are intended to match 9.x